docs/user-guide/replicasets.md

```diff
@@ -36,6 +36,9 @@ provides declarative updates to pods along with a lot of other useful features.
 Therefore, we recommend using Deployments instead of directly using Replica Sets, unless
 you require custom update orchestration or don't require updates at all.
 
+This actually means that you should never need to write a replica set object:
```

Comment:
This isn't quite true. See the text above.

If you use choose to use a Deployment, then you should never need to directly manage the replica sets it creates.